UefiPayloadPkg/UefiPayloadEntry/UefiPayloadEntry.h

```diff
@@ -27,6 +27,7 @@
 #include <IndustryStandard/MemoryMappedConfigurationSpaceAccessTable.h>
 #include <Guid/SmBiosTableHob.h>
 #include <Guid/AcpiTableHob.h>
+#include <Library/BlParseLib.h>

```

Comment:
Please avoid including BlParseLib.h because the UniversalPayloadEntry doesn't link to this lib. You can duplicate the GET_BOOTLOADER_PARAMETER macro.